### PR TITLE
User management - Add icon to user groups

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Review by marcoambrosini — APPROVED** [review]

**Review by kesselb — APPROVED** [review]

File: apps/settings/src/views/Users.vue [hunk 1]
```diff
@@ -81,7 +81,8 @@
 					:key="group.id"
 					:exact="true"
 					:title="group.title"
-					:to="{ name: 'group', params: { selectedGroup: encodeURIComponent(group.id) } }">
+					:to="{ name: 'group', params: { selectedGroup: encodeURIComponent(group.id) } }"
+					icon="icon-group">
```
[the remark this answers is not in the document]
Comment (artonge) [reply apps/settings/src/views/Users.vue]: Confused by the state changes, Am I a bit late to the party ?

[the remark this answers is not in the document]
Comment (szaimen) [reply apps/settings/src/views/Users.vue]: I disaggree on this one. Having no icons here looks pretty bad thats why I've added them here:
image

[the remark this answers is not in the document]
Comment (artonge) [reply apps/settings/src/views/Users.vue]: Let's summon @nextcloud/designers

[the remark this answers is not in the document]
Comment (szaimen) [reply apps/settings/src/views/Users.vue]: BTW the new PR is here: https://github.com/nextcloud/server/pull/28166

**Pull request closed without merging**